app.py

```diff
@@ -662,6 +695,19 @@ def generate_radio_options(selected_date, day_type='Weekday', daterange_type=0):
                 for period
                 in TIMEPERIODS[TIMEPERIODS['day_type'] == day_type]['period']]
 
+
+@app.callback(Output('step2', 'children'), 
```

Comment:
Create the div ids at the top so that the linter can detect if the variables exist, rather than hard-coding strings in each Input/Output. It will save you from inscrutable errors when you make a typo.